mxnet/training/docker/1.8.0/py3/Dockerfile.cpu

```diff
@@ -11,6 +11,7 @@ ARG MX_URL=https://aws-mx-pypi.s3-us-west-2.amazonaws.com/1.8.0/aws_mx-1.8.0-py2
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
 ARG SMDEBUG_VERSION=0.9.4
+ARG OPENSSL_VERSION=1.1.1g
```

Comment:
Keeping this as is. UL16 do not have `libssl1.1` package. ref: https://packages.ubuntu.com/search?suite=default&section=all&arch=i386&keywords=libssl&searchon=names

Reply:
In that case, could you change this to `OPENSSL_VERSION=1.1.1j`?

Reply:
Added.